synfig-studio/plugins/lottie-exporter/lottie-exporter.py

```diff
@@ -95,6 +95,9 @@ def gen_html(file_name):
         (None)
     """
 
+    # Take only the file name, to take relative file path
+    store_file_name = file_name.split("/")[-1]
```

Comment:
It might worth to use os.path.basename() here. See - https://docs.python.org/3/library/os.path.html#os.path.basename

Reply:
Yes, Thank you for suggestion! 
Fixed now.